src/views/HomeView.vue

```diff
@@ -1,39 +1,69 @@
 <template>
-  <div class="container" id="landingContainer">
-    <div class="span-image">
-      <img src="/public/images/field.jpg" class="landingImage">
-    </div>
-    <div id="textOverImage">
-      <div class="col titleArea">
-        <h1>Forward<br><span class="logoProgress">Progress</span></h1>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col aboutRow">
-          <router-link to="about" class="aboutButton infoButton">Who We Are</router-link>
+  <div id="landingContainer">
+    <div class="featured-section">
+      <div class="row header3 mb-3">
+        <p>Leading the Way Forward in Football Analysis</p>
       </div>
+      <b-row align-h="center">
+        <b-col cols="4">
+          <b-carousel
+            id="landing-page-article-carousel"
+            :interval="4000"
+            fade
+            controls
+            indicators
+            background="#ababab"
+            img-width="100%"
+            img-height="400px"
+            v-model="currentSlide"
+          >
+            <b-carousel-slide v-for="post in carouselSlides" v-bind:key="post.id"
+              :img-src="post.coverImageURL"
+            />
+          </b-carousel>
+          <div class="slider-article-description">
+            <div class="slider-description-content">
+              <div class="slider-title">{{ currentSlidePost.title }}</div>
+              <div class="slider-subtitle">{{ currentSlidePost.subTitle }}</div>
+            </div>
+          </div>
+        </b-col>
+        <b-col cols="4">
+          <div class="latest-container">
+            <div class="red-title">
+              LATEST
+            </div>
+            <div>
+              <div v-for="post in otherLatestPosts" v-bind:key="post.id" class="latest-post">
+                <div class="post-title">
+                  {{ truncate(post.title, 'title') }}
+                </div>
+                <div class="post-subtitle">
+                  {{ truncate(post.subTitle, 'subtitle') }}
+                </div>
+                <div v-if="!isLastLatestPost(post)" class="divider"/>
+              </div>
+            </div>
+          </div>
+        </b-col>
+      </b-row>
     </div>
-    <div class="row">
-      <div class="col aboutRow aboutRowSubHeader">
-        <p class="subHeader">A web based SaaS platform for front office NFL executives.</p>
-      </div>
-    </div>
-    <div class="row header3">
-      <p>Leading the Way Forward in Football Analysis</p>
-    </div>
-    <div class="featureSpotlight">
+    <div
+      class="featureSpotlight"
+      style="border-top:1px dotted black;padding-top:30px;"
```

Comment:
I didn't touch this in this PR, but I would like to remove inline styles in favor of using CSS classes, however small the styling may be. It is easier to keep track of if they are CSS classes instead